EasyPost/Models/API/Options.cs

```diff
@@ -114,8 +120,14 @@ public class Options : EasyPostObject
         [JsonProperty("by_drone")]
         public bool? ByDrone { get; set; }
 
+        [JsonProperty("carrier_branded")]
+        public bool? CarrierBranded { get; set; }
+
+        [JsonProperty("carrier_instructions")]
+        public Dictionary<string, object?>? CarrierInstructions { get; set; } // hash
+
         [JsonProperty("carrier_insurance_amount")]
-        public string? CarrierInsuranceAmount { get; set; }
+        public string? CarrierInsuranceAmount { get; set; } // number
```

Comment:
Should the comment be `any`? 
```suggestion
        public string? CarrierInsuranceAmount { get; set; } // any
```